andela/troupon
# Ch setup troupon cloudinary account 128055597

#### What does this PR do?

This PR updates the deal fixture - initial_data.json with a deal image URL that points to images stored on Troupon's Cloudinary account rather than a team member's account.

#### Description of Task to be completed?

This task consists of:
- Creating a new account on Cloudinary in Troupon's name called "andela-troupon".
- Moving images from the previous Cloudinary account to the new account created.
- Update initial_data.json with the new information.

#### How should this be manually tested?
- Visit the troupon site.
- Right click one of the deals image and select "inspect element".
- Check the src attribute of the img tag and confirm you can see "andela-troupon" in the URL.

#### What are the relevant pivotal tracker stories?

#128055597

#### Screenshots:
![screen shot 2016-08-17 at 4 53 06 pm](https://cloud.githubusercontent.com/assets/7263503/17743347/50f3d24c-649b-11e6-8cd2-690900d8c4c2.png)
